src-docs/src/views/selectable/selectable_example.js

```diff
@@ -142,21 +145,20 @@ export const SelectableExample = {
           />
         </Fragment>
       ),
-      props: {
-        EuiSelectable,
-        EuiSelectableOptionProps,
-        EuiSelectableOptionsList,
-        EuiSelectableSearchProps,
-      },
+      props,
       demo: <SelectableSearch />,
       snippet: `<EuiSelectable
   aria-label="Searchable example"
+  options={[
+    { label: 'A', searchableLabel: 'Alpha' },
+    { label: 'B', searchableLabel: 'Beta' },
```

Comment:
Nit: Putting `searchableLabel` in the snippet make it seem required. I would show both options:
```suggestion
    { label: 'Alpha' },
    { label: 'B', searchableLabel: 'Beta' },
```

Reply:
Or create multiple snippets.

Reply:
Just one option works for me! 👍

Reply:
https://github.com/elastic/eui/pull/6074/commits/fc001b21dddc0f74bb44fef4564c9d42ba353f15